src/library/process.js

```diff
@@ -35,8 +35,19 @@ export const processMessage = async (AWS, region, service, account,
         ':serv': { S: `${service}` },
       },
     },
-  );
+    );
+    
+  // CR: Mickey: we should think about whether it's preferable to use graphQl here
+  //    to pluck just vendor data from the account record.
+
+  // CR: Mickey: this code would be more readable if instead of repeated itemExists calls
+  //    we had a helper (like lodash/get) that protected against errors when trying to find
+  //    a nested property.  so this could be if(_get(msgBody, 'context.user.accountId')) { ... }
+  //    as a simple condition
 
+  // CR: Mickey: is there a case where context.user.accountId is not passed and we
+  //    want the request to continue as if it was?  what about context.user.userId?
+  //    are there any values on context that we should throw an error without?
```

Comment:
@MickeyMcc There should ideally never be a case at least when coming from dashboard that context will not have these values. However, there will be cases where a worker in a service created using this framework may not need those values. An example can be a service working off enrollmentId which will not care about userId.